app/utils/popoverMultiple.js

```diff
@@ -0,0 +1,48 @@
+let timerArray = [];
```

Comment:
Do we need a separate popover utility for this?
Could we just modify existing popover instead of adding new utility?